storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -147,6 +149,12 @@ static bool reopen_log_files(lsn_t desired_lsn) {
       srv_read_only_mode, log.m_encryption_metadata, files, format,
       creator_name, log_flags, log_uuid);
 
+  if (desired_lsn == 0) {
+    if (res != Log_files_find_result::FOUND_VALID_FILES)
+      return false;
+    else
+      return true;
+  }
```

Comment:
```suggestion
  if (desired_lsn == 0) 
    return (res == Log_files_find_result::FOUND_VALID_FILES);
```

Reply:
Here we abort the execution of the function earlier.
Is there any harm on not populating the metadata we do in the lines below?

Reply:
@rahulmalik87 ^

Reply:
population the medata now.